test/unit/org/apache/cassandra/index/sai/utils/AbstractPrimaryKeyTester.java

```diff
@@ -154,7 +154,7 @@ DecoratedKey makeKey(TableMetadata table, Object...partitionKeys)
         return table.partitioner.decorateKey(key);
```

Comment:
nit: It might be nicer to use `key = table.partitionKeyType.decomposeUntyped(partitionKeys[0]);` above on 153 to avoid some unnecessary `String` conversions?